protocol-designer/src/components/modules/AdditionalItemsRow.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { FLEX_ROBOT_TYPE } from '@opentrons/shared-data'
+import {
+  OutlineButton,
+  Flex,
+  JUSTIFY_SPACE_BETWEEN,
+  ALIGN_CENTER,
+  DIRECTION_COLUMN,
+  LabeledValue,
+  SPACING,
+  SlotMap,
+} from '@opentrons/components'
+import { WASTE_CHUTE_SLOT } from '../../constants'
+import gripperImage from '../../images/flex_gripper.svg'
+import styles from './styles.css'
+
+interface AdditionalItemsRowProps {
+  handleAttachment: () => void
+  isEquipmentAdded: boolean
+  name: 'gripper' | 'wasteChute'
+}
+
+export function AdditionalItemsRow(
+  props: AdditionalItemsRowProps
+): JSX.Element {
+  const { handleAttachment, isEquipmentAdded, name } = props
+  const { i18n, t } = useTranslation()
+
+  return (
+    <Flex
+      justifyContent={JUSTIFY_SPACE_BETWEEN}
+      marginBottom="0.5rem"
+      height="6rem"
+    >
+      <Flex flexDirection={DIRECTION_COLUMN}>
+        <h4 className={styles.row_title}>
+          {name === 'gripper'
+            ? i18n.t('modules.additional_equipment_display_names.gripper')
+            : i18n.t('modules.additional_equipment_display_names.wasteChute')}
+        </h4>
+        <AdditionalItemImage
+          //  TODO(jr, 9/13/23): update this image to the waste chute asset
+          src={gripperImage}
+          alt={name === 'gripper' ? 'Flex Gripper' : 'Waste Chute'}
+        />
+      </Flex>
+      <div
+        className={styles.module_col}
+        style={{ marginLeft: SPACING.spacing32 }}
+      >
+        {isEquipmentAdded && name === 'gripper' ? (
+          <LabeledValue
+            label="Model"
+            value={i18n.t(`modules.model_display_name.gripperV1`)}
+          />
+        ) : null}
+      </div>
+
+      {name === 'wasteChute' && isEquipmentAdded ? (
```

Comment:
nit
for consistency 

```suggestion
      {isEquipmentAdded && name === 'wasteChute' ? (
```